### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by gmzych**
First few tests which I've made recently

### CODE REVIEW

**Review by DawidBastek — CHANGES_REQUESTED** [review]
Good job! 👍  Few small fixes and we are ready to go to the next level 😎

File: tests/base_tests.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            touch = TouchAction(self.driver)
+            touch.press(x=535, y=2022).move_to(x=535, y=56).release().perform()
+        self.driver.find_element_by_xpath('/hierarchy/android.widget.FrameLayout/android.view.ViewGroup/android.widget.FrameLayout[2]/android.widget.RelativeLayout/android.widget.LinearLayout/android.widget.ListView/android.widget.LinearLayout[14]').click()
+
+    def test_find_element_and_go_back(self):
+        touch = TouchAction(self.driver)
+        for x in range(2):
+            touch.press(x=508, y=2022).move_to(x=514, y=511).release().perform()
+            self.driver.find_element_by_id('/hierarchy/android.widget.FrameLayout/android.view.ViewGroup/android.widget.FrameLayout[2]/android.widget.RelativeLayout/android.widget.LinearLayout/android.widget.ListView/android.widget.LinearLayout[8]/android.widget.TextView').click()
+        mapelement = self.driver.find_element_by_xpath('/hierarchy/android.widget.FrameLayout/android.view.ViewGroup/android.widget.FrameLayout[2]/android.widget.FrameLayout/android.widget.FrameLayout/android.widget.FrameLayout/android.widget.RelativeLayout[2]')
```
[the remark this answers is not in the document]
Comment (gmzych) [reply tests/base_tests.py]: sure :), I will be using snake_case for variables, methods, functions and CamelCase for classes

File: tests/base_tests.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        element = self.driver.find_element_by_xpath('/hierarchy/android.widget.FrameLayout/android.view.ViewGroup/android.widget.FrameLayout[2]/android.widget.FrameLayout/android.widget.FrameLayout/android.widget.FrameLayout/android.widget.RelativeLayout[2]/android.widget.ImageView')
+        assert element.is_displayed()
+
+    def test_screen_scroll(self):
+        touch = TouchAction(self.driver)
+        touch.press(x=535, y=2022).move_to(x=535, y=56).release().perform()
+        for x in range(5):
+            touch = TouchAction(self.driver)
+            touch.press(x=535, y=2022).move_to(x=535, y=56).release().perform()
+        self.driver.find_element_by_xpath('/hierarchy/android.widget.FrameLayout/android.view.ViewGroup/android.widget.FrameLayout[2]/android.widget.RelativeLayout/android.widget.LinearLayout/android.widget.ListView/android.widget.LinearLayout[14]').click()
```
[the remark this answers is not in the document]
Comment (gmzych) [reply tests/base_tests.py:21]: Ok, it's corrected in another PR

**Review by DawidBastek — DISMISSED** [review]

**Review by DawidBastek — APPROVED** [review]
Looks good for this stage!
One thing, you could install `pylint` for your project. If you're using PyCharm here is the link:
https://plugins.jetbrains.com/plugin/11084-pylint

**Pull request merged**